demo/gallery/vat-bob.js

```diff
@@ -52,6 +53,44 @@ function makeBobMaker(E, log) {
           );
           return amountP;
         },
+        async buyFromCorkBoard(handoffSvc, dustPurseP) {
+          const { pixelIssuer, dustIssuer } = await E(gallery).getIssuers();
+          const collect = makeCollect(E, log);
+          const boardP = E(handoffSvc).grabBoard('MeetPoint');
+          const contractHostP = E(boardP).lookup('contractHost');
+          const buyerInviteP = E(boardP).lookup('buyerSeat');
+          const buyerSeatP = E(contractHostP).redeem(buyerInviteP);
+
+          const pixelPurseP = E(pixelIssuer).makeEmptyPurse('purchase');
+          E(buyerSeatP).offer(dustPurseP);
+          const dustRefundP = E(dustIssuer).makeEmptyPurse('dust refund');
+          await collect(buyerSeatP, pixelPurseP, dustRefundP, 'bob option');
+
+          const exclusivePayment = await E(pixelIssuer).getExclusiveAll(
+            pixelPurseP,
+          );
+
+          const { useRightPayment } = await E(gallery).split(exclusivePayment);
+
+          const useRightIssuer = E(useRightPayment).getIssuer();
```

Comment:
I think we should use the useRightIssuer from the gallery separately rather than getting it from the useRightPayment. In both cases, we're trusting the gallery, but getting the issuer separately allows us to do a double check

Reply:
done.